cmd/fleetctl/gitops.go

```diff
@@ -201,7 +201,10 @@ func gitopsCommand() *cli.Command {
 				// so we can bail if any of the referenced labels wouldn't exist
 				// after this run (either because they'd be deleted, never existed
 				// in the first place).
-				labelsUsed := getLabelUsage(config)
+				labelsUsed, err := getLabelUsage(config)
+				if err != nil {
+					return err
+				}
```

Comment:
Took the opportunity to update this to validate that multiple `label_*` keys aren't included at once (see below)

Reply:
Nice!